beacon-chain/node/config.go

```diff
@@ -65,6 +65,24 @@ func configureSafeSlotsToImportOptimistically(cliCtx *cli.Context) error {
 	return nil
 }
 
+func configureMevBoostCircuitBreakerValues(cliCtx *cli.Context) error {
```

Comment:
maybe we should name it form `configureMevBoostCircuitBreakerValues` to`configureBuilderCircuitBreaker`